File: app/src/main/java/com/filemanager/fileexplorer/myfile/cleanmaster/free/adapter/GirlAdapter/GridViewHolder.java
package com.filemanager.fileexplorer.myfile.cleanmaster.free.adapter.GirlAdapter;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.recyclerview.widget.RecyclerView;

import com.demo.example.R;


public class GridViewHolder extends RecyclerView.ViewHolder {
    ImageView imageView;
    ImageView img_check;
    TextView textView;

    public GridViewHolder(View view) {
        super(view);
        this.textView = (TextView) view.findViewById(R.id.text_grid);
        this.imageView = (ImageView) view.findViewById(R.id.icon_grid);
        this.img_check = (ImageView) view.findViewById(R.id.img_check);
    }
}

File: app/src/main/java/com/demo/filemanager/adapter/GirlAdapter/GridViewHolder.kt
package com.demo.filemanager.adapter.GirlAdapter

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.demo.filemanager.R


class GridViewHolder(view: View) : RecyclerView.ViewHolder(view) {
    var imageView: ImageView
    var img_check: ImageView
    var textView: TextView

    init {
        textView = view.findViewById<View>(R.id.text_grid) as TextView
        imageView = view.findViewById<View>(R.id.icon_grid) as ImageView
        img_check = view.findViewById<View>(R.id.img_check) as ImageView
    }
}

